open_city_profile/tests/conftest.py

```diff
@@ -39,26 +40,26 @@ def superuser():
 
 @pytest.fixture
 def gql_client():
-    gql_client = GraphQLClient(schema)
+    gql_client = GraphQLClient(schema, format_error=GraphQLView.format_error)
```

Comment:
you could subclass `GraphQLClient` and bake in your custom format_error in the subclass to avoid specifying it every time. hm, could also have it as a fixture and let other fixtures extend it and override the `.user` attribute. basically, something can be done to avoid this multiple duplicate lines 😅

Reply:
Ended up just having a function that returns the formatted client. We need to refactor that conftest anyways in order to make tests quicker and to not do anything unnecessary stuff between tests.

Reply:
OK, good enough for the temp solution 👍